src/profile/profile.styles.jsx

```diff
@@ -0,0 +1,27 @@
+import styled, { css } from 'react-emotion';
+
+export const TabContainer = styled('div')`
+  display: flex;
+  margin-bottom: 10px;
+`;
+
+export const TabHeader = styled('div')`
+  border-radius: 0.4em;
+  background-image: linear-gradient(-225deg, #22E1FF 0%, #1D8FE1 48%, #625EB1 100%);

```

Comment:
Do you think we'd benefit from having a `Themes` folder and a file called `colors.js` that we can import to our style sheets? I've seen that being used in some big boilerplates

Reply:
`Themes` may also store some images, icons, `metrics` (like heights, widths, font sizes etc that we use a lot)

Reply:
Yeah, I think we should. Was just kind of playing around with colors / showing off styles.